### ISSUE TITLE
SemanticUSL pose.txt issues

### ISSUE THREAD

**Issue by haamoon** [unmannedlab/LiDARNet#4]
Hey!

Thanks for the great work and releasing the code and dataset. I realized that there are a couple of minor issues in the `poses.txt` files. I was wondering if you can help me fix them?

1) For some sequences, number of rows in `poses.txt` is 1 or 2 lines fewer than the number of velodyne scans in the same sequence. The is the case `'00', '02', '04', '05', '06', '07', '08', '09', '10', '11', '13', '14', '16', '17', '18', '19', '20', '22', '23', '24', '25', '26', '27', '28', '29', '30', '31', '33', '34', '35', '36', '37', '38', '39', '40', '41', '42', '43', '44', '45', '46', '47', '48', '49', '51', '53', '55', '57', '59', '61', '63', '65', '67', '68', '69', '70'`
2) The first entry in `poses.txt` has very large values and seems to be wrong. This is the case for `'16', '19', '59', '68', '69', '70'`

How would you suggest to fix these issues?

**Comment by maskjp** [comment]
Hi,@haamoon,

That always happens at the beginning of the sequence. You can copy the close rows to overlap them. 

I hope this helps!

Best wishes!

**Comment by haamoon** [comment]
How about the size mismatch? Shall I duplicate the first line or the last line?

**Comment by maskjp** [comment]
Hi, @haamoon:

There's a mismatch? Could you tell me which sequences are they?

Yes, you can try to remove front or end scans of the sequences. Because the lidar is at a very high frequency, you will not lose much accuracy or data. 

Thanks!

Best wishes!

Peng Jiang

**Comment by haamoon** [comment]
Yes, I have listed the sequences with mismatch in my first post (also listed below). Thanks for the recommendation, I will try it.

`'00', '02', '04', '05', '06', '07', '08', '09', '10', '11', '13', '14', '16', '17', '18', '19', '20', '22', '23', '24', '25', '26', '27', '28', '29', '30', '31', '33', '34', '35', '36', '37', '38', '39', '40', '41', '42', '43', '44', '45', '46', '47', '48', '49', '51', '53', '55', '57', '59', '61', '63', '65', '67', '68', '69', '70'`
